### DISCUSSION TITLE
r/city-china-henan-luoyang-laojunshan

### DISCUSSION THREAD

**Discussion by giscus** [ops-coffee/blog#43] [Announcements]
# r/city-china-henan-luoyang-laojunshan

游记｜远赴人间惊鸿宴，一睹人间盛世颜

https://blog.ops-coffee.cn/r/city-china-henan-luoyang-laojunshan

**Comment by 7wate** [comment]
我去过多次老君山，仅次黄山了 ～

**Comment by varzy** [comment]
牛肉汤对味了！

**Comment by zwz1437** [comment]
可以添加友链吗？已将您添加为友链，向您学习，非常感谢。我的博客：https://tanscp.com

**Comment by LessSugarClub** [comment]
嗯？原来是老乡啊～

**Comment by Gimorocun** [comment]
老乡，初来上海工作，见到家乡人在他处开枝散叶，享自在人生，实在欣喜，又给人勇气
老君山景色真的很棒，等下过雪，2月份天气明朗时，能看到山与雪融合一体的场景，推荐大家有空爬一下老君山

**Reply by ops-coffee** [reply]
哈哈，很高兴，茫茫人海中还能遇到老乡，预祝工作顺利
